src/core/annotation.js

```diff
@@ -66,10 +66,12 @@ AnnotationFactory.prototype = /** @lends AnnotationFactory.prototype */ {
   /**
    * @param {XRef} xref
    * @param {Object} ref
+   * @param {string} uniquePrefix
+   * @param {Object} idCounters
    * @returns {Annotation}
    */
-  create: function AnnotationFactory_create(xref, ref,
-                                            uniquePrefix, idCounters) {
+  create: function AnnotationFactory_create(xref, ref, uniquePrefix,
+                                            idCounters) {
```

Comment:
Nit: Since you're not making any functional changes here, please revert this to avoid unnecessarily changing the `blame` of these lines.